bin/setup_base

```diff
@@ -15,7 +21,7 @@ fi
 
 pip3 install -r etc/requirements.txt
 
-DEF_IFACE=`route -n | fgrep UG | awk '{print $8}'`
+DEF_IFACE=`route -n | grep -E '^0\.0\.0\.0' | awk '{print $8}'`
```

Comment:
Use 'egrep \sUG\s' -- Haoli had to do this on another PR as well, so just want to have a consistent solution. @henry54809

Reply:
Updated.